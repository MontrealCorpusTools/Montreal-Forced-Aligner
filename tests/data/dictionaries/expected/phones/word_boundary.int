1 nonword
2 begin
3 end
4 internal
5 singleton
6 nonword
7 begin
8 end
9 internal
10 singleton
11 begin
12 end
13 internal
14 singleton
15 begin
16 end
17 internal
18 singleton
19 begin
20 end
21 internal
22 singleton
