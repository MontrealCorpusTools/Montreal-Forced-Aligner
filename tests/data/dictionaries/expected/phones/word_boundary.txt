sil nonword
sil_B begin
sil_E end
sil_I internal
sil_S singleton
spn nonword
spn_B begin
spn_E end
spn_I internal
spn_S singleton
phonea_B begin
phonea_E end
phonea_I internal
phonea_S singleton
phoneb_B begin
phoneb_E end
phoneb_I internal
phoneb_S singleton
phonec_B begin
phonec_E end
phonec_I internal
phonec_S singleton
